### PR TITLE
[FRONT-4] Rename purchases as subscriptions

### PR DESCRIPTION
**Pull request by juhah**
Renamed instances of "purchase" in the UI as "subscriptions" (and in some internal code though not all).

### CODE REVIEW

File: app/src/app/Userpages.jsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ const UserpagesRouter = () => ([
     <Route exact path={routes.streams.index()} component={StreamListViewAuth} key="StreamListView" />,
     <Route exact path={routes.streams.preview()} component={StreamLivePreview} key="StreamLivePreview" />,
     <Route exact path={routes.transactions()} component={TransactionListAuth} key="TransactionList" />,
-    <Route exact path={routes.purchases()} component={PurchasesPageAuth} key="PurchasesPage" />,
+    <Route exact path={routes.subscriptions()} component={PurchasesPageAuth} key="PurchasesPage" />,
```
Comment (juhah) [inline app/src/app/Userpages.jsx:51]: This route might need to be changed on external components in shared menus

File: app/src/userpages/components/Header/index.jsx [hunk 2]
```diff
@@ -60,22 +60,22 @@ const Header = ({
                     </div>
                     <div className={styles.tabs}>
                         <Tab to={routes.streams.index()}>
-                            <Translate value="userpages.header.streams" />
+                            <Translate value="general.streams" />
```
Comment (mondoreale) [inline app/src/userpages/components/Header/index.jsx:63]: ❤️

**Comment by juhah** [comment]
> Are we replacing all purchase-related copy with subs? If so, then there's a modal – PurchaseModal, usePurchase hook and a bunch of tests that haven't been migrated. Does it apply to them too or too much hassle atm?

This should cover all of the UI and references to them in docs.. We could/should rename all code as well for completeness' sake but yeah, too much hassle for now 😅

**Comment by mondoreale** [comment]
Cool with me!

**Pull request merged**